lib/sly-completion.el

```diff
@@ -204,20 +204,24 @@ COMPLETIONS is a list of propertized strings."
                                                completion))
            (add-text-properties 0
                                 (length completion)
-                                `(sly--annotation
-                                  ,(format "%s %5.2f%%"
-                                           classification
-                                           (* score 100))
-                                  sly--suggestion
-                                  ,suggestion)
+                                `(sly--classification ,classification
+                                  sly--score ,score
+                                  sly--suggestion ,suggestion)
                                 completion)
 
            collect completion into formatted
            finally return (list formatted nil)))
 
 (defun sly-completion-annotation (completion)
-  "Grab the annotation of COMPLETION, a string, if any"
-  (get-text-property 0 'sly--annotation completion))
+  "Grab the classification and score of COMPLETION, if exists, and generate a string for display. The string is a concatenation of either if exists. Return empty string if neither classification or score is available."
```

Comment:
Super long line.  Press `M-q` here.  The first line should be a complete sentence ending in period.  

But I suggest not spending time detailing the implementation or what an annotation entails, if might have another
for in the future, so I suggest.

```
Compute annotation of COMPLETION as a string.
Return the empty string if none exists.
```